src/tools/r2rdump/R2RReader.cs

```diff
@@ -87,8 +87,7 @@ public unsafe R2RReader(string filename)
                 if (peReader.HasMetadata)
                 {
                     var mdReader = peReader.GetMetadataReader();
-                    R2RMethods = new R2RMethod[mdReader.MethodDefinitions.Count];
-
+                    
                     int i = 0;
```

Comment:
If `i` is only used for the loop, I recommend moving the `int i = 0` declaration to the for loop.